interface/i18n/mist.nl.i18n.json

```diff
@@ -57,12 +57,12 @@
             },
             "help": {
                 "label": "Help",
-                "reportBug": "Report an issue on Github"
+                "reportBug": "Rapporteer een probleem op Github"
             }
         },
         "errors": {
-            "nodeConnect": "Verbinden naar node niet mogelijk? Zie de logboek bestanden voor meer informatie:",
-            "nodeStartup": "Het lijkt erop dat de node niet kan worden opgestart, heeft u er al een aan staan? Is deze de database aan het upgraden?",
+            "nodeConnect": "Verbinden met node niet mogelijk? Zie de logboek bestanden voor meer informatie:",
+            "nodeStartup": "Het lijkt erop dat de node niet kan worden opgestart, heb je er al een draaien? Is deze de database aan het upgraden?",
```

Comment:
Similar to your other pull there is the question of you/u (formal/informal). Not sure what the guidelines are.

Perhaps a neutral version like:

> "Het lijkt erop dat de node niet kan worden opgestart, **draait er al een**? Is deze de database aan het upgraden?"